src/Features/VisualBasic/Portable/Completion/CompletionProviders/ImportCompletionProvider/ExtensionMethodImportCompletionProvider.vb

```diff
@@ -0,0 +1,36 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Collections.Immutable
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.Completion.Providers
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Shared.Extensions.ContextQuery
+Imports Microsoft.CodeAnalysis.Text
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
+    Friend NotInheritable Class ExtensionMethodImportCompletionProvider
+        Inherits AbstractExtensionMethodImportCompletionProvider
+
+        Protected Overrides ReadOnly Property GenericSuffix As String
+            Get
+                Return "(Of )"
```

Comment:
is that how we generally do it.  we don't do `(Of ...)`?

Reply:
You are right. Fixed